### PR TITLE
Big Clean Up 🧴🧽✨        

### PR DESCRIPTION
**Pull request by johnxgaehner**
homework finished, what you think?
- add BEM naming convention
- add profile page
- add bookmarks page
- add basic create page layout

### CODE REVIEW

**Comment by AlexJanow** [comment]
Love the style!

File: styles/_base.css [hunk 1]
```diff
@@ -13,3 +13,133 @@ html {
...
+
+.question-card {
+  margin: 2em;
+  padding: 2em;
+  border: 2px solid black;
+  box-shadow: var(--universal-box-shadow);
+  position: relative;
+}
+
+.question-card:nth-child(3n + 0) {
```
Comment (namirsab) [inline styles/_base.css:84]: You don't need the `+0`. 3n would be enough

File: styles/_base.css [hunk 2]
```diff
@@ -13,3 +13,133 @@ html {
...
+}
+
+.question-card__answer-text {
+  display: none;
+  margin: 2em 0 0 0;
+}
+
+.question-card__answer-section:active .question-card__answer-text {
+  display: block;
+}
```
Comment (namirsab) [inline styles/_base.css:129]: Good job in here using the `:active` pseudo-class to show the answer on tap!

**Review by namirsab — CHANGES_REQUESTED** [review]
Hey @johnxgaehner , really good job! Just move that body closing tag ;)

**Pull request merged**